specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/playbookConfigurations.json

```diff
@@ -0,0 +1,511 @@
+            "$ref": "./examples/PlaybookConfigurations/ValidatePlaybookConfiguration_example.json"
+          }
+        },
+        "tags": [ "PlaybookConfigurations" ],
+        "description": "Validate the playbook configuration model before create or update. Any validation errors are returned to the client.",
+        "operationId": "PlaybookConfigurations_Validate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PlaybookConfigurationName"
+          },
+          {
+            "$ref": "#/parameters/PlaybookConfigurationInBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PlaybookConfigurationValidationStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PlaybookConfigurationList": {
+      "description": "List of playbook configurations response.",
+      "required": [ "value" ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The list of playbook configurations under the given scope.",
+          "items": {
+            "$ref": "#/definitions/PlaybookConfiguration"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "PlaybookConfiguration": {
+      "type": "object",
+      "description": "The playbook configuration resource.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Playbook configuration data",
+          "$ref": "#/definitions/PlaybookConfigurationProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "PlaybookConfigurationProperties": {
+      "type": "object",
+      "description": "The playbook configuration data.",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The playbook configuration description."
+        },
+        "isEnabled": {
+          "type": "boolean",
+          "description": "Indicates whether the playbook configuration is enabled."
+        },
+        "metadata": {
```

Comment:
>metadata [](start = 9, length = 8)

Should this be called "status" rather than "metadata"?

Reply:
No, this is the metadata that describes the playbookConfiguration. Currently the meta-data describes who updated it and when (additional info will be added in the future)

Reply:
ok

---
In reply to: [281738432](https://github.com/Azure/azure-rest-api-specs/pull/5548#discussion_r281738432) [](ancestors = 281738432)